packages/plugin-vue/README.md

```diff
@@ -51,7 +51,7 @@ export interface Options {
 }
 ```
 
-## Example for passing options to `@vue/compiler-dom`:
+## Example for passing options to `@vue/compiler-sfc`:
```

Comment:
```suggestion
## Example for passing options to `vue/compiler-sfc`:
```

Let's use `vue/compiler-sfc` here to avoid confusion (it's equivalent to `@vue/compiler-sfc` but more recommended).